contracts/SiphonDepositor.sol

```diff
@@ -109,35 +126,33 @@ contract SiphonDepositor {
      *      Along with a pro rata amount of AURA tokens
      */
     function getReward() external {
+        uint256 balBefore = cvx.balanceOf(address(this));
         (, , , address crvRewards, , ) = booster.poolInfo(pid);
         IBaseRewardPool(crvRewards).getReward(address(this), false);
+        uint256 balAfter = cvx.balanceOf(address(this));
+        // Increment total farmed amount
+        farmedTotal += (balAfter - balBefore);
     }
 
     /**
      * @dev How much AURA you will get for rAURA
+     *      rAURA can be redeemed at a rate determined by the totalSupply
+     *      of rAURA and totalAmount of AURA the depositor has farmed.
      */
     function getAmountOut(uint256 _amount) public view returns (uint256) {
-        uint256 totalSupply = cvx.totalSupply();
-        uint256 INIT_MINT_AMOUNT = cvx.INIT_MINT_AMOUNT();
-        // TODO: this is internal on the Aura token...
-        uint256 minterMinted = 0;
-        uint256 reductionPerCliff = cvx.reductionPerCliff();
-        uint256 totalCliffs = cvx.totalCliffs();
-        uint256 EMISSIONS_MAX_SUPPLY = cvx.EMISSIONS_MAX_SUPPLY();
-
-        uint256 emissionsMinted = totalSupply - INIT_MINT_AMOUNT - minterMinted;
-        uint256 cliff = emissionsMinted.div(reductionPerCliff);
-
-        uint256 amount;
-        if (cliff < totalCliffs) {
-            uint256 reduction = totalCliffs.sub(cliff).mul(5).div(2).add(700);
-            amount = _amount.mul(reduction).div(totalCliffs);
-            uint256 amtTillMax = EMISSIONS_MAX_SUPPLY.sub(emissionsMinted);
-            if (amount > amtTillMax) {
-                amount = amtTillMax;
-            }
-        }
-        return amount;
+        uint256 totalSupply = rCvx.totalSupply();
+        return (_amount * farmedTotal) / totalSupply;
+    }
+
+    /**
+     * @dev Transfer ERC20 tokens to recipient
+     */
+    function transferTokens(
```

Comment:
?

Reply:
Unlikely this will stay like this but for now this function is used to transfer tokens to the "bridge" contract. Likely we will just replace this with transferToBridge but need to account for multiple bridge contracts which haven't been written yet. Just added a TODO: comment to clarify this.